app/(app)/p/[frameId]/[handler]/[[...query]]/route.ts

```
@@ -1,83 +1,97 @@
-import type { BaseConfig, BaseStorage } from '@/lib/types'
-import { FrameError } from '@/sdk/error'
-import templates from '@/templates'
-import { eq } from 'drizzle-orm'
-import ms from 'ms'
-import { notFound } from 'next/navigation'
-import type { NextRequest } from 'next/server'
-
-export const dynamic = 'force-dynamic'
-export const dynamicParams = true
-export const fetchCache = 'force-no-store'
+import { NextRequest } from 'next/server';
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)